recipes/khiopsdriver-gcs/conda_build_config.yaml

```diff
@@ -0,0 +1,13 @@
+---
+cxx_compiler:
+  - gxx # [linux]
+  - clangxx # [osx]
+  - vs2022 # [win]
```

Comment:
```suggestion
```

Do not set these manually unless you are changing the defaults, which you are not doing here.